dnsapi/dns_octodns.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env sh
+
+# This API is a wrapper for other API so that you
+# are able to put your TXT record to multiple providers.
+# The functionality is in the responsibility of the respective API
+# therefore please check if your used APIs work.
+# e.g. for using NS1 and AWS Route53, use:
+# OCTODNS_PROVIDERS=nsone_aws
```

Comment:
why?
why we need your api to load other providers ?

Reply:
That is because this is just a wrapper for other APIs. The idea of octodns is have dns as code and provide the possibility to use multiple dns providers for your domains. And when using multiple providers, in my example NS1 and AWS Route53, we need to put the acme TXT Record on both providers at once (or we get a 50% chance of failing to create the let's encrypt certificate). 
Let's take the domain swisstxt.org for example. When you query the NS records, you get
```;; ANSWER SECTION:
swisstxt.org.		3600	IN	NS	ns-544.awsdns-04.net.
swisstxt.org.		3600	IN	NS	dns4.p07.nsone.net.
swisstxt.org.		3600	IN	NS	ns-1660.awsdns-15.co.uk.
swisstxt.org.		3600	IN	NS	ns-181.awsdns-22.com.
swisstxt.org.		3600	IN	NS	ns-1490.awsdns-58.org.
swisstxt.org.		3600	IN	NS	dns2.p07.nsone.net.
swisstxt.org.		3600	IN	NS	dns3.p07.nsone.net.
swisstxt.org.		3600	IN	NS	dns1.p07.nsone.net.
```
That is why I created this wrapper API where you can define which providers are in use and the acme script will automatically put the TXT records to all listed providers (and delete them after use).